lib/routes/account.js

```diff
@@ -820,45 +821,26 @@ module.exports = (log, db, mailer, Password, config, customs, signinUtils, push)
         }
       },
       handler: async function (request) {
-        var auth = request.auth
-        var uid
+        const auth = request.auth
+        let uid, scope
         if (auth.strategy === 'sessionToken') {
           uid = auth.credentials.uid
+          scope = { contains: () => true }
```

Comment:
Just to understand, if the user calls /account/profile with a sessionToken, we say "all scopes" are good. This would be used by the settings page?

Reply:
Yes, exactly.